### PR TITLE
Add alerts for missing dcos-storage objects

### PR DESCRIPTION
**Pull request by bbannier**

### CODE REVIEW

**Review by jdef — COMMENTED** [review]
MISSING may or may not be a transient condition: it's a guesstimate that DSS makes based on currently available data reported by Mesos - which may not 100% reflect reality. Ideally alerts would only fire if these metrics experienced an increase over some time window (i.e. an increase remained "stable" for a period of 5m or longer).

My concern is that these alerts, as they're coded here, might flap a bit and people would learn to ignore them. I like the idea of multi-window error budget burn rates, as described here:

https://developers.soundcloud.com/blog/alerting-on-slos

... but we probably don't have time to land that very soon. I'm wondering if there's a simpler variant of such rules where we could only fire an alert after some period of time has elapsed (5m?) where `count` is greater than zero?

**Comment by bbannier** [comment]
@jdef, I added a `5m` window as suggested by you. While I agree that multi-window thresholds would be great, this is not something we currently do and likely requires some more thinking (e.g., subqueries -- which could be helpful here -- have only landed in prometheus-2.7.0, but we bundle an earlier version). Let's punt on this for now.

**Review by jdef — APPROVED** [review]

**Comment by jdef** [comment]
PTAL @gracedo

File: rules/dcos-storage.yml [hunk 1]
```diff
@@ -39,3 +39,27 @@ groups:
...
+    annotations:
+      summary: DC/OS storage service has at least one missing volume
+  - alert: "Device MISSING"
+    expr: count(dss_obj_devices_missing) > 0
+    for: 5m
+    labels:
+      severity: warning
+      service: dcos-storage
+    annotations:
+      summary: DC/OS storage service has at least one missing volume
```
Comment (gracedo) [inline rules/dcos-storage.yml]: `volume` -> `device` ?

Comment (jdef) [reply rules/dcos-storage.yml]: yep, we'll change it

Comment (jdef) [reply rules/dcos-storage.yml]:
```suggestion
      summary: DC/OS storage service has at least one missing device
```

**Review by philipnrmn — APPROVED** [review]

**Pull request merged**